index.html

```diff
@@ -18,21 +18,40 @@
     />
   </head>
   <body>
-    <div class="container" id="header"></div>
+    <div class="container" id="header">
+      <header>
+        <img class="logo" src="images/logo.png" alt="logo" />
+        <nav class="topnav">
+          <ul>
+            <li class="menuitem"><a href="#about">About</a></li>
```

Comment:
In the CSS you reference `#menuItem` but not `.menuitem` or `.menu_item`... You probably want these to be in sync?